flight/Modules/Stabilization/stabilization.c

```diff
@@ -744,7 +757,7 @@ static void stabilizationTask(void* parameters)
 									rateDesiredAxis[YAW] = pid_apply(&pids[PID_ATT_YAW], axis_lock_accum[YAW], dT);
 									rateDesiredAxis[YAW] = bound_sym(rateDesiredAxis[YAW], settings.MaximumRate[YAW]);
 
-									actuatorDesiredAxis[YAW] = pid_apply_setpoint(&pids[PID_RATE_YAW],  rateDesiredAxis[YAW],  gyro_filtered[YAW], dT);
+									actuatorDesiredAxis[YAW] = pid_apply_setpoint(&pids[PID_RATE_YAW], NULL, rateDesiredAxis[YAW], gyro_filtered[YAW], dT);
```

Comment:
AFAIK coordinated flight is plane-only, so I did not apply it there. Same applies to Virtualbar.

That big ass switch statement gets iterated over every axis, so it's being applied in the other sections (like *_RATE, *_ATTITUDE, etc.).